tests/unit/views/ont/OntHeronRuns.spec.js

```diff
@@ -12,29 +12,31 @@ describe('OntHeronRuns.vue', () => {
 
     wrapper = mount(OntHeronRuns, {
       localVue,
+      mocks: {
+        $apollo: {
+          queries: {
+            runs: {
+              refetch: jest.fn()
+            },
+          },
+        },
+      },
```

Comment:
I think the UI pagination branch removed the refetch runs, depending on the order of merges might need checking